extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts

```diff
@@ -125,16 +127,23 @@ abstract class HdfsDialogModelBase<T extends DialogProperties> {
 		return new ClusterController(this.props.url, this.props.auth, this.props.username, this.props.password, true);
 	}
 
-}
-
-export class MountHdfsDialogModel extends HdfsDialogModelBase<MountHdfsProperties> {
-	private credentials: {};
-
-	constructor(props: MountHdfsProperties) {
-		super(props);
+	protected async createAndVerifyControllerConnection(): Promise<ClusterController> {
+		// We pre-fetch the endpoints here to verify that the information entered is correct (the user is able to connect)
+		let controller = this.createController();
+		let response: IEndPointsResponse;
+		try {
+			response = await controller.getEndPoints();
+			if (!response || !response.endPoints) {
+				throw new Error(localize('mount.hdfs.loginerror1', "Login to controller failed"));
```

Comment:
login error? copy-paste?

Reply:
I have 2 cases, 1 where I know the error message and one without. Hence the 2 messages loginerror1 and 2